src/Type/EntityTypeManagerGetStorageDynamicReturnTypeExtension.php

```diff
@@ -2,19 +2,30 @@
 
 namespace mglaman\PHPStanDrupal\Type;
```

Comment:
I did not change the namespace of this class to avoid BC break in case someone already extend it.

What do you think about: 
- copy/paste it to `mglaman\PHPStanDrupal\Type\EntityStorage` namespace
- make it extends the new class in `mglaman\PHPStanDrupal\Type\EntityStorage` namespace
- deprecate it

or do you think it's ok to do the BC break?

Reply:
I don't care about backward compatibility with this library. The only namespace that I would care around BC are Rules, but none of them are opt-in yet, they just run. A decision I'd like to change.

Break the BC! 😄